src/javascripts/modules/ExpandList.js

```diff
@@ -0,0 +1,32 @@
+import debounce from 'lodash.debounce'
+
+export default class ExpandList {
+  constructor(el) {
+    this.el = el
+    this.tagArray = Array.from(this.el.querySelectorAll('.tag-list__item'))
+    this.btnArray = Array.from(this.el.querySelectorAll('.tag-list__btn'))
+
+    this.bindEvents()
+  }
+
+  bindEvents() {
+    this.btnArray.forEach(btn => btn.addEventListener('click', this.expand))
+    window.addEventListener('resize', debounce(this.resize, 250))
+  }
+
+  //for expanding tag-list__items when clicking tag-list__btns
+  expand = (e) => {
+    let isExpanded = e.currentTarget.getAttribute('aria-expanded') === 'true'
+    e.currentTarget.setAttribute('aria-expanded', !isExpanded)
+    
+    let btnIndex = this.btnArray.indexOf(e.currentTarget)
+    this.tagArray[btnIndex].style.visibility = isExpanded ? 'hidden' : 'visible'
+  }
+
+  //for reverting all open or closed tags to visible when resizing browser window
+  resize = () => {
```

Comment:
This part works well (going from large to small), but when going from small to large (this would happen on an iPad), all the tags open. If that's not your intent, then you'd need to add another matchMedia statement. for `min-width: 960px` (and make this one `959px`). Also might want to make the units match the rem units you have in sass.